third_party/nlbuild-autotools/repo/scripts/mkversion

```diff
@@ -161,7 +161,11 @@ gitversion() {
 				fi
 
 			else
-				gitversion=`${PRINTF} "g${head}"`
```

Comment:
Turned out I was falling down the rabbit hole. The most recent version makes no changes to nlbuild-autotools.